packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -120,7 +120,7 @@
 					:dialect="getArgument('sqlDialect')"
 					:isReadOnly="isReadOnly"
 					:rows="getArgument('rows')"
-					@valueChanged="valueChangedDebounced"
+					@update:modelValue="valueChangedDebounced"
```

Comment:
can you also add tests for the new sql editor behavior that's in scope here?

Reply:
this is code from master, Set overhaul had an old copy of this line, this revert change

Reply:
we still need tests
1. to prevent this bug again
2. to test the new behavior

Reply:
I would say as this revert change to master we should merge this bugfix,
no new behavior to sql editor was added here, 
I agree that tests should be added, but in my opinion it is outside of this PR scope

Reply:
at least an e2e test to prevent this bug again.. otherwise it's not fixed.. As policy, a bug is not considered fixed until tests have been added to prevent it. and should not be hard to add quickly.
and create followup for the other tests